kyuubi-server/src/main/scala/org/apache/kyuubi/engine/trino/TrinoProcessBuilder.scala

```diff
@@ -113,18 +108,16 @@ class TrinoProcessBuilder(
     if (commands == null) {
       super.toString
     } else {
-      Utils.redactCommandLineArgs(conf, commands).map {
-        case arg if arg.contains(ENGINE_TRINO_CONNECTION_PASSWORD.key) =>
-          s"${ENGINE_TRINO_CONNECTION_PASSWORD.key}=$REDACTION_REPLACEMENT_TEXT"
-        case arg if arg.contains(ENGINE_TRINO_CONNECTION_KEYSTORE_PASSWORD.key) =>
-          s"${ENGINE_TRINO_CONNECTION_KEYSTORE_PASSWORD.key}=$REDACTION_REPLACEMENT_TEXT"
-        case arg if arg.contains(ENGINE_TRINO_CONNECTION_TRUSTSTORE_PASSWORD.key) =>
-          s"${ENGINE_TRINO_CONNECTION_TRUSTSTORE_PASSWORD.key}=$REDACTION_REPLACEMENT_TEXT"
-        case arg => arg
-      }.map {
-        case arg if arg.startsWith("-") => s"\\\n\t$arg"
-        case arg => arg
-      }.mkString(" ")
+      redactConfValues(
+        Utils.redactCommandLineArgs(conf, commands),
+        Set(
+          ENGINE_TRINO_CONNECTION_PASSWORD,
+          ENGINE_TRINO_CONNECTION_KEYSTORE_PASSWORD,
+          ENGINE_TRINO_CONNECTION_TRUSTSTORE_PASSWORD).map(_.key))
```

Comment:
simply use 
```suggestion
        Set(
          ENGINE_TRINO_CONNECTION_PASSWORD.key,
          ENGINE_TRINO_CONNECTION_KEYSTORE_PASSWORD.key,
          ENGINE_TRINO_CONNECTION_TRUSTSTORE_PASSWORD.key))
```
otherwise would broken if we want to add a string literal later

Reply:
No problem. Let me change the style here and for other process builders.